### PR TITLE
Feature: In Water Display

### PR DESCRIPTION
**Pull request by Nealkitt**
## What
Displays if the player is In Water for use with the Prismarine Blade/Shark Scale Armor, mostly useful on Stranded

<details>
<summary>Images</summary>

image
image
</details>

## Changelog New Features
+ Added In-Water Display. - Stella
    * Useful when using a Prismarine Blade in Stranded Mode.

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/features/misc/InWaterDisplay.kt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    @SubscribeEvent
+    fun onTick(event: LorenzTickEvent) {
+        playerInWater = Minecraft.getMinecraft().thePlayer.isInWater
+    }
+
+    @SubscribeEvent
+    fun onRenderOverlay(event: GuiRenderEvent.GuiOverlayRenderEvent) {
+        if (!isEnabled()) return
+
+        val text = "In Water: " + if (playerInWater) "§aTrue" else "§cFalse"
```
Comment (j10a1n15) [inline src/main/java/at/hannibal2/skyhanni/features/misc/InWaterDisplay.kt]: Why don't you just directly check the player inwater here

Comment (Nealkitt) [reply src/main/java/at/hannibal2/skyhanni/features/misc/InWaterDisplay.kt]: huge oversight by me, sorry i'm new to this

Comment (j10a1n15) [reply src/main/java/at/hannibal2/skyhanni/features/misc/InWaterDisplay.kt]: No worries

Comment (martimavocado) [reply src/main/java/at/hannibal2/skyhanni/features/misc/InWaterDisplay.kt]: i don't think there's a need to check every frame instead of every tick

Comment (ItsEmpa) [reply src/main/java/at/hannibal2/skyhanni/features/misc/InWaterDisplay.kt]: player inwater is already a boolean that is updated every tick

Comment (martimavocado) [reply src/main/java/at/hannibal2/skyhanni/features/misc/InWaterDisplay.kt]: oh in that case it's fine, i had no idea how the check worked

**Comment by hannibal002** [comment]
please add more info to the changelog: why is this needed?

**Comment by Nealkitt** [comment]
> please add more info to the changelog: why is this needed?

Added that it's for stranded players

**Comment by martimavocado** [comment]
changelog should look like this :3
```
+ Added In Water Display. - Stella
    * Useful while using Prismarine Blade in Stranded.
```

**Review by CalMWolfs — APPROVED** [review]

**Comment by hannibal002** [comment]
also jani does not like white guis

**Pull request merged**